src/cobalt.rs

```diff
@@ -230,6 +313,10 @@ pub fn build(config: &Config) -> Result<()> {
         }
 
         let mut context = doc.get_render_context(&posts_data);
+        let mut site = HashMap::new();
+        site.insert("data".to_owned(), Value::Object(data_map.clone()));
```

Comment:
Is there a reason to create `site` twice?

We'll be adding more information to `site` over time and it'd be nice to not duplicate the logic.

Reply:
no, not necessary.